dolphinscheduler-standalone-server/src/main/docker/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-FROM eclipse-temurin:8-jre
+FROM eclipse-temurin:8-jdk
```

Comment:
Why did you change this?

Reply:
Because the JRE image does not have the installation of the JavaC command, the Javatask needs to be compiled before execution
![image](https://github.com/apache/dolphinscheduler/assets/127465317/e3030111-a16a-4a1f-8cdb-87a9847f0ec1)

Reply:
I got it. Thanks.